server/database/models/admin-model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const adminSchema = new Schema({
+  email: {
+    type: String,
+    lowercase: true,
+    unique: true,
+    required: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  userName: {
+    type: String,
+    required: true,
+  },
+});
+module.exports = mongoose.model('adminSchema', adminSchema);
```

Comment:
change schema name in the first attribute to be like this 
```suggestion
module.exports = mongoose.model('admin', adminSchema);
```